src/components/Combobox/__stories__/combobox.stories.mdx

```diff
@@ -129,7 +129,7 @@ Default combo box can be used without dialog or as part of the layout.
 
 ### Comobox inside a dialog
 
-Use this for combo box that triggered by button.
+Use this for Combobox that triggered by button.
 
 <Canvas>
   <Story name="Comobox inside a dialog">
```

Comment:
Typo

Reply:
it was intentional)

Reply:
No no, `Comobox` lol

Reply:
Aahh, now I see it :D
Will fix, thanks!